src/main/java/com/dreammaster/scripts/ScriptNuclearControl.java

```diff
@@ -303,5 +301,65 @@ public void loadRecipes() {
                 .itemOutputs(getModItem(IC2NuclearControl.ID, "ItemVanilliaKit", 1, 0, missing)).duration(1600).eut(2)
                 .addTo(sAssemblerRecipes);
 
+        GT_Values.RA.stdBuilder() // Color upgrade
+                .itemInputs(
+                        new ItemStack(Items.dye, 1, 1),
+                        new ItemStack(Items.dye, 1, 2),
+                        new ItemStack(Items.dye, 1, 4),
+                        GT_OreDictUnificator.get(OrePrefixes.circuit, Materials.Advanced, 1L))
+                .itemOutputs(getModItem(IC2NuclearControl.ID, "ItemUpgrade", 1, 1, missing)).duration(400).eut(480)
```

Comment:
please use notation like TierEU.RecipeHV and 20 * SECONDS

also applies to the others

Reply:
zu langsam. ahhhh

Reply:
Oh missed that

Reply:
I don’t know exactly what voltage it will set, if it’s literally 512 then for early tiers there will be problems with losses

Reply:
No you need to use the variable hv tier. The value is only need if you use values between tiers

Reply:
its 15/16th of tier not 512.
these ones https://github.com/GTNewHorizons/GT5-Unofficial/blob/b43e65f3784a7978656a4e2a72a2734b37d7b735/src/main/java/gregtech/api/enums/TierEU.java#L25